src/emqx_sys.erl

```diff
@@ -94,7 +94,17 @@ sysdescr() -> emqx_app:get_description().
 %% @doc Get sys uptime
 -spec(uptime() -> string()).
 uptime() ->
-    gen_server:call(?SYS, uptime).
+    {TotalWallClock, _} = erlang:statistics(wall_clock),
```

Comment:
Put this format function to a side-effect free func, so we can unit test the formatter